types/vote.go

```diff
@@ -13,7 +13,8 @@ import (
 )
 
 const (
-	nilVoteStr string = "nil-Vote"
+	absentVoteStr string = "absent"
```

Comment:
```suggestion
	absentVoteStr string = "Vote{absent}"
```
I'd still format this so people recognise that it is a vote